charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml

```diff
@@ -150,6 +150,8 @@ spec:
         args:
           - --loglevel={{ .Values.log.level }}
           - --drivemgrendpoint={{ .Values.drivemgr.grpc.server.endpoint }}
+          - --usenodeannotation={{ .Values.feature.usenodeannotation }}
```

Comment:
These changes are only needed for `loopback manager`. Can you make them `loopback manager` specific only? Can we also read `node name` through environment variable?

Reply:
@mishoyama 
Yep, sure.
Do we have a rule about things we pass as parameters and others with env? `nodename` is already passed as a parameter in the node container, so why we should use a different approach in drivemgr?

Reply:
No, we don't have such rule. But if we can discover something just using standard k8s env vars why do we need to have extra argument?

Reply:
I agree, but `nodename` is KUBE_NODE_NAME env and it's not the standard k8s env. It's made from spec.
Should I change the node container to use env instead?